Template/milestone/table.php

```diff
@@ -131,6 +128,9 @@
     <tr>
         <th colspan="4" class="total"><?= t('Total time tracking') ?></th>
         <td<?php if ($editable): ?> colspan="2"<?php endif ?>>
+
+            <strong><?= count($milestone) ?></strong> <?= strtolower(t('Tasks')) ?>
```

Comment:
What do you think about moving this number of tasks directly into the progress bar l.148?

Reply:
That would be great too.